src/main/java/games/strategy/triplea/delegate/BaseEditDelegate.java

```diff
@@ -46,7 +46,11 @@ public static boolean getEditMode(final GameData data) {
     if (!(editMode instanceof Boolean)) {
       return false;
     }
-    return (Boolean) editMode;
+    return (boolean) editMode;
```

Comment:
If possible in the future, please avoid making incidental changes unrelated to the core of the PR when it is this large (or at least call them out in a review).  Since I had ~1800 lines to review, and I thought they were just methods being moved around, I wasn't scrutinizing each line.  Now I'm second guessing myself if I need to go back and re-review the 800 or so lines I've already hit (just kidding :smile:).

Agreed that there is no danger here that wasn't already present as the automatic unboxing would have the same effect (and we've ensured `editMode` is not `null`).